ui/components/app/cancel-button/cancel-button.component.js

```diff
@@ -0,0 +1,64 @@
+import { Tooltip } from '@material-ui/core';
```

Comment:
Nit: the `.component.js` naming convention is from when we used to use the "container" / "component" pattern, with the Redux hooks in the "container" and everything else in the "component". Now that we're using React Hooks instead, I don't think we need a `.component.js` suffix anymore.

Reply:
✅